### PR TITLE
feat: add arb owner and arb gas info actions

### PR DESCRIPTION
**Pull request by spsjvc**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -62,6 +62,46 @@ jobs:
       - name: Test
         run: yarn test:unit
 
+  test-integration:
```
[the remark this answers is not in the document]
Comment (GreatSoshiant) [reply .github/workflows/build-test.yml]: Thanks Dragisa! resolved!

File: src/index.ts [hunk 2]
```diff
@@ -68,6 +70,9 @@ import { prepareKeyset } from './prepareKeyset';
 import * as utils from './utils';
 
 export {
+  arbOwnerPublicActions,
+  arbGasInfoPublicActions,
+  createRollup,
```
[the remark this answers is not in the document]
Comment (GreatSoshiant) [reply src/index.ts:75]: Which ones do you mean @spsjvc ? You mean all of them?

File: src/decorators/arbOwnerPublicActionsUpgradeExecutor.integration.test.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+const client = createPublicClient({
+  chain: orbitLocal,
+  transport: http(),
+}).extend(arbOwnerPublicActions);
+
+// Client for arb gas info public actions
+const client2 = createPublicClient({
+  chain: orbitLocal,
+  transport: http(),
+}).extend(arbGasInfoPublicActions);
```
[the remark this answers is not in the document]
Comment (GreatSoshiant) [reply src/decorators/arbOwnerPublicActionsUpgradeExecutor.integration.test.ts]: Wow this is amazing Dragisa! I didn't know that!

File: src/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (spsjvc) [reply src/index.ts:75]: no, just `createRollup`. it doesn't exist anymore

**Review by GreatSoshiant — APPROVED** [review]
LGTM!

**Pull request merged**